app/root.tsx

```
@@ -1,25 +1,41 @@
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <link
           rel="stylesheet"
           as="style"
           crossOrigin="true"
           href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard-dynamic-subset.min.css"
         />
+        {googleTrackingId && (
+          <Fragment>
+            <script async src={`https://www.googletagmanager.com/gtag/js?id=${googleTrackingId}`}></script>
+            <script
+              dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)